cmake/utilities.cmake

```diff
@@ -249,7 +249,7 @@ function(vcpkg_target_add_warning_options TARGET)
     endif()
 endfunction()
 
-function(vcpkg_target_add_sourcelink target)
+function(vcpkg_target_add_sourcelink TARGET)
```

Comment:
This is the wrong way, it should be `vcpkg_target_add_sourcelink target`, and then fixed below

Reply:
```suggestion
function(vcpkg_target_add_sourcelink target)
```